<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDDoserWindow</class>
 <widget class="QMainWindow" name="WBSDDoserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>B2 doser system</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>751</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="rightB2Test">
         <property name="text">
          <string>Activate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Right B2 test</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="leftB2Test">
         <property name="text">
          <string>Activate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Left B2 test</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="leftB2SlowTest">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Activate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="leftSlowLab">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Left B2 slow test</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rightSlowLab">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Right B2 slow test</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="rightB2SlowTest">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Activate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>leftB2Test</tabstop>
  <tabstop>rightB2Test</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
